ex2/ex2.c

```diff
@@ -8,7 +8,24 @@
 
 int main(void)
 {
-    // Your code here 
-    
+    // Your code here
+    FILE *fp;
+
+    fp = fopen("text.txt", "w");
```

Comment:
good work jasmine!, great logic and make sure you clean up your code a bit by eliminating redundant white spaces